package.json

```diff
@@ -3,20 +3,39 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@emotion/react": "^11.11.1",
+    "@emotion/styled": "^11.11.0",
     "@headlessui/react": "^1.7.17",
     "@heroicons/react": "^2.0.18",
+    "@mui/icons-material": "^5.14.8",
+    "@mui/material": "^5.14.8",
+    "@react-spring/web": "^9.7.3",
+    "@reduxjs/toolkit": "^1.9.5",
+    "@tailwindcss/forms": "^0.5.6",
```

Comment:
Honestly you can have better option then using tailwind forms with all material UI stuff, I think it would be better even to use material forms with formik or react hook form

Reply:
I have modified the majority of the forms to function with Formik and Yup.